internal/service/eks/addon_data_source.go

```diff
@@ -70,6 +75,7 @@ func dataSourceAddonRead(ctx context.Context, d *schema.ResourceData, meta inter
 	d.Set("addon_version", addon.AddonVersion)
 	d.Set("arn", addon.AddonArn)
 	d.Set("created_at", aws.TimeValue(addon.CreatedAt).Format(time.RFC3339))
+	d.Set("configuration_values", addon.ConfigurationValues)
```

Comment:
We should include AccTest modifications for the data source as well.

Reply:
Added a new test for data soource